test/tests/azurestack_tags_map/input/main.tf

```diff
@@ -6,6 +6,10 @@ terraform {
   }
 }
 
+provider "azurestack" {
```

Comment:
This test becomes broken in some versions.
Adding this fixes it. (Maybe a new provider version requires it?).